c2h/include/c2h/catch2_main.h

```diff
@@ -67,6 +67,6 @@ int main(int argc, char* argv[])
 
   set_device(device_id);
 #  endif // THRUST_DEVICE_SYSTEM == THRUST_DEVICE_SYSTEM_CUDA
-  return session.run(argc, argv);
+  return session.run();
```

Comment:
The args are already handled by the call to `session.applyCommandLine(argc, argv)` above on line 62.